src/test/resources/debug-etc/config.yml

```diff
@@ -1,29 +1,82 @@
-
-server:
-  applicationContextPath: /
-  adminContextPath: /
-  applicationConnectors:
-    - type: http
-      # Replace with port number unique for this service
-      port: 20000
-  adminConnectors:
-    - type: http
-      # Replace with port number unique for this service
-      port: 20001
-
 #
```

Comment:
This `config.yml` is copied by `start-env.sh` to `etc/`. The idea is that `start-env.sh` will set things up for you so you can run `start-service.sh` to start debugging. You'll notice that this config file refers to locations in your local `data/` subdirectory.